README.md

```diff
@@ -9,7 +9,7 @@
 
 ## Live Demo
 
-[Live Demo Link](https://raw.githack.com/samlorlah/awesome-books/milestone-1/index.html)
+[Live Demo Link](https://samlorlah.github.io/awesome-books/)
```

Comment:
- [ ] [OPTIONAL] Good job so far 👏 Your README looks great. However, there is one important section that I would suggest you add. It is the `screenshot section`. This is where you add the screenshot of the landing page of your web app. This is very important as it gives the user a quick preview of what your app looks like 😉 . Please consider adding it too to make your README more professional.